package.json

```diff
@@ -29,7 +29,7 @@
     "clean": "rimraf dist/*",
     "format": "prettier --write \"src/**/*.ts\" \"src/**/*.js\"",
     "lint": "tslint --project tsconfig.json",
-    "build": "npm run lint && npm run test && npm run clean && tsc",
+    "build": "npm run lint && npm run test -- -u && npm run clean && tsc",
```

Comment:
Adding this here is a bit risky. Test snapshots are part of project code and any changes to them should be resolved before running the building. Otherwise they either won't be saved (the build process should only run tests without silently overwriting any of the project code) or may sneak in unnoticed to the main branch during the next build. I've had that kind of situations in the past. :(

Please remove `-- -u` from here, then run `npm test -- -u` locally on your branch, then commit any resulting changes.

Reply:
I just noticed it. I did it because I could not pass some snapshots, and I have found this solution on StackOverflow.

*I will hardly revert my two previous commits (this, and the next one)* mainly because I am not experienced with testing at all. I understand that I should cover my case with testes, by since I have no idea how to do it right, I am trying my best.

Reply:
All right, don't worry too much about testing if you have problems, I'll see if it can be fixed next week. There's a reason codecov isn't among _required_ status checks. :)